zebra/rtadv.h

```diff
@@ -145,6 +159,8 @@ extern void rtadv_stop_ra_all(void);
 extern void rtadv_cmd_init(void);
 extern void zebra_interface_radv_disable(ZAPI_HANDLER_ARGS);
 extern void zebra_interface_radv_enable(ZAPI_HANDLER_ARGS);
+extern void rtadv_add_prefix(struct zebra_if *zif, struct prefix_ipv6 *p);
+extern void rtadv_delete_prefix(struct zebra_if *zif, struct prefix *p);
```

Comment:
Could prefix pointers `p` be `const` here?

Reply:
No problem.  will do